uni/lib/view/bug_report/widgets/text_field.dart

```diff
@@ -60,9 +62,9 @@ class FormTextField extends StatelessWidget {
                     labelStyle: Theme.of(context).textTheme.bodyMedium,
                   ),
                   controller: controller,
-                  validator: (String? value) {
-                    if (value == null || value.isEmpty) {
-                      return isOptional ? null : emptyText;
+                  validator: (value) {
+                    if (value!.isEmpty) {
```

Comment:
What if the value is null (e.g. widget not fully loaded)?

Reply:
Does the `!` operator not check for null values?

Reply:
The ! (bang) operator actually unwraps the nullabe object into a non nullable one. It will raise an exception if the value is null